### PR TITLE
chore: clean up LW7

### PR DESCRIPTION
**Pull request by fsansalvadore**
## What kind of change does this PR introduce?

Wraps and cleans up the website from LW mode:
- bring favicon back to normal
- bring logo back to normal
- remove LW7 Announcement banner
- minor mobile improvements on LW7 landing page
- update loading animation in studio
- rename useMobileViewport to useBreakpoint

## Additional context

Favicon and logo updated, announcement banner removed:
Schermata 2023-04-17 alle 16 09 00

Loading animation updated:

https://user-images.githubusercontent.com/25671831/232510304-e6063f97-5cff-4ea8-989e-39347aaa7ddf.mov

Mobile improvements on LW landing page:
Schermata 2023-04-17 alle 14 22 50

### CODE REVIEW

File: apps/docs/components/Favicons.tsx [hunk 1]
```diff
@@ -22,15 +22,15 @@ const Favicons = () => {
       {/* prettier-ignore */}
       <link rel="apple-touch-icon-precomposed" sizes="152x152" href={`${basePath}/favicon/apple-touch-icon-152x152.png`}/>
       {/* prettier-ignore */}
-      <link rel="icon" type="image/png" href={`${basePath}/favicon/favicon-196x196.png" sizes="196x196`}/>
```
Comment (MildTomato) [inline apps/docs/components/Favicons.tsx:25]: sure about this @fsansalvadore ?

File: apps/www/hooks/useBreakpoint.tsx [hunk 2]
```diff
@@ -2,7 +2,7 @@ import { useEffect, useState } from 'react'
 import { useWindowSize } from 'react-use'
 import { isBrowser } from '~/lib/helpers'
 
-export function useMobileViewport(breakpoint = 768) {
+export function useBreakpoint(breakpoint = 768) {
```
Comment (saltcod) [inline apps/www/hooks/useBreakpoint.tsx:5]: wonder if this should go in packages/common?

File: apps/www/components/Favicons.tsx [hunk 3]
```diff
@@ -22,15 +22,15 @@ const Favicons = () => {
       {/* prettier-ignore */}
       <link rel="apple-touch-icon-precomposed" sizes="152x152" href={`${basePath}/favicon/apple-touch-icon-152x152.png`}/>
       {/* prettier-ignore */}
-      <link rel="icon" type="image/png" href={`${basePath}/favicon/favicon-196x196.png" sizes="196x196`}/>
+      <link rel="icon" type="image/png" href={`${basePath}/favicon/favicon-196x196.png`}/>
       {/* prettier-ignore */}
```
Comment (MildTomato) [inline apps/www/components/Favicons.tsx:26]: checking this as well. the sizes attribute.

File: studio/components/head/Favicons.tsx [hunk 4]
```diff
@@ -22,15 +22,15 @@ const Favicons = () => {
       {/* prettier-ignore */}
       <link rel="apple-touch-icon-precomposed" sizes="152x152" href={`${basePath}/favicon/apple-touch-icon-152x152.png`}/>
       {/* prettier-ignore */}
-      <link rel="icon" type="image/png" href={`${basePath}/favicon/favicon-196x196.png" sizes="196x196`}/>
```
Comment (MildTomato) [inline studio/components/head/Favicons.tsx:25]: same again

**Review by saltcod — APPROVED** [review]

File: apps/docs/components/Favicons.tsx (hunk 1 above)
Comment (fsansalvadore) [reply apps/docs/components/Favicons.tsx:25]: hmmm it was showing an error in console about those but I can't reproduce it anymore... so I'll revert this for now

Comment (fsansalvadore) [reply apps/docs/components/Favicons.tsx:25]: Sorry, there actually was something wrong. Just fixed the quotes now.

File: studio/components/head/Favicons.tsx (hunk 4 above)
Comment (fsansalvadore) [reply studio/components/head/Favicons.tsx:25]: The quotes were off actually, fixed it 👍

**Pull request merged**